app.js

```diff
@@ -86,7 +86,6 @@ const env = opts.env || ['dev'];
 const hostname = opts.host;
 const port = opts.port || 10240;
 const staticDir = opts.static || 'static';
-const archivedVersions = opts.archivedVersions;
```

Comment:
This was really weird, because I remember it being used somewhere. Any idea when it stopped being used?

Reply:
With webpack we've stopped doing this. This is as intended :)